exercises/C-comparison-operators/exercise.js

```diff
@@ -7,14 +7,14 @@
 
 let studentCount = 16;
 let mentorCount = 9;
-let moreStudentsThanMentors; // finish this statement
+let moreStudentsThanMentors = true; // finish this statement
```

Comment:
Exercise is telling You to use the comparison operator to prove that, for example, there are more students than mentors. For example:
let moreStudentsThanMentors = studentCount > mentorCount

Please check this link: https://www.w3schools.com/js/js_comparisons.asp